docs/capabilities.md

```diff
@@ -250,6 +250,28 @@ CAP_ODBC_UNBIND_BATCH | Set to 'yes' to force the Tableau native ODBC protocol t
 CAP_ODBC_UNBIND_EACH | Set to 'yes' to force the Tableau native ODBC protocol to unbind and deallocate columns individually, which may improve stability. | yes | yes 
 CAP_ODBC_UNBIND_PARAMETERS_BATCH | Set to ‘yes’ to unbind all parameters in a single batch operation. | yes | yes 
 
+### How to read ODBC metadata is determined by capabilities
```

Comment:
Can you give this topic its own page, similar to [auth modes](https://tableau.github.io/connector-plugin-sdk/docs/auth-modes): metadata-enumeration.

Basically a general page about what this overall topic is, something like: When connecting to a data source Tableau needs to know hierarchy, and enumerate entities.  When selecting a table or connecting to an existing saved connection the column level info needs to be queried as well.

Then have an ODBC section with this content

Reply:
ok good idea, will do